README.md

```diff
@@ -1064,19 +1063,40 @@ vars:
   dbt_project_evaluator:
     # set your chained views threshold to 8 instead of 4
     chained_views_threshold: 8
-    # update the number of records inserted from the graph from 10,000 to 500 to reduce query size
-    insert_batch_size: 500
 ```
 </details>
 
 <details>
-<summary><b>Warehouse Specific Variables</b></summary>
+<summary><b>Execution Variables</b></summary>
 
 | variable    | description | default     |
 | ----------- | ----------- | ----------- |
-| `max_depth_dag` | limits the number of looped CTEs when computing the DAG end-to-end for BigQuery and Databricks/Spark compatibility | 9 |
+| `max_depth_dag` | limits the maximum length between nodes calculated in `int_all_dag_relationships` | 9 |
+| `generate_all_dag_paths` | boolean to limit recursive SQL for Postgres, Snowflake, Redshift, Duckdb  | see below |
+| `insert_batch_size` | number of records inserted per batch when unpacking the graph into models | 10000 |
+
+__Note on max_depth_dag and generate_all_dag_paths__
+
+The default behavior for limiting the relationships calculated in the `int_all_dag_relationships` model differs depending on your adapter. 
+
+- For Bigquery & Spark/Databricks, `int_all_dag_relationships`, the maximum DAG path length is set by the `max_depth_dag` variable, which is defaulted to 9. This is because these adapters do not currently support recursive SQL. 
```

Comment:
```suggestion
- For Bigquery & Spark/Databricks the maximum distance between two nodes in your DAG, calculated in `int_all_dag_relationships`, is set by the `max_depth_dag` variable, which is defaulted to 9. So by default, `int_all_dag_relationships` contains a row for every path less than or equal to 9 nodes in length between two nodes in your DAG. This is because these adapters do not currently support recursive SQL. 
```